GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -157,8 +157,19 @@ public class Glia {
     }
 
     /// Clear visitor session
-    public func clearVisitorSession() {
+    ///
+    /// - Parameter completion: Completion handler.
+    ///
+    /// - Important: Note, that in case of ongoing engagement, `clearVisitorSession` must be called after ending engagement,
+    /// because `GliaError.clearingVisitorSessionDuringEngagementIsNotAllowed` will occur otherwise.
+    ///
+    public func clearVisitorSession(_ completion: (Result<Void, Error>) -> Void) {
```

Comment:
Since `clearVisitorSession` has changed its type from `() -> Void` to `((Result<Void, Error>) -> Void ) -> Void` that will become a breaking change for integrators. Also completion is not `@escaping` at the moment, meaning if we are to do some async work there in future, we will be forced be compiler to deprecate it and introduce `@escaping` completion. Also what benefit do we get right now from it if it is not escaping? So to address both potential problems I suggest to add `@escaping` and make compilation optional.

```suggestion
    public func clearVisitorSession(_ completion: @escaping ((Result<Void, Error>) -> Void)? = nil) {
```

Does it make sense @EgorovEI @gersonnoboa @yurii-glia @rasmustautsglia?

Reply:
You are totally right.

Reply:
85e0de910eba66310c6f8ab45a11b11e991df6ca